src/features/BroadcastTransaction/stateFactory.tsx

```diff
@@ -70,7 +70,10 @@ const BroadcastTxConfigFactory: TUseStateReducerFactory<State> = ({ state, setSt
       cb();
     } catch (err) {
       console.debug(`[BroadcastTx] ${err}`);
-      displayToast(toastTemplates.failedTransaction);
```

Comment:
Are we tossing out toasts from here on?

Reply:
I just found it to be unnecesary here when we already have an error message. We still use them elsewhere. If you prefer I can re-add?